tutorials/auto_scheduler/tune_network_x86.py

```diff
@@ -138,7 +140,9 @@ def get_network(name, batch_size, layout="NHWC", dtype="float32", use_sparse=Fal
     if use_sparse:
         from tvm.topi.sparse.utils import convert_model_dense_to_sparse
 
-        mod, params = convert_model_dense_to_sparse(mod, params, random_params=True)
+        mod, params = convert_model_dense_to_sparse(
+            mod, params, bs_r=sparse_block_size[0], bs_c=sparse_block_size[1], random_params=True
```

Comment:
Let's keep this tutorial simple.

Suggest to move the sparse_block_size setting inside this function, for example directly set the block size here.

So that we can avoid to modify the function parameters and add `sparse_block_size = (8, 1)` outside.

Reply:
Done.